deepmd/pt/model/model/ener_model.py

```diff
@@ -95,3 +95,40 @@ def forward_lower(
         else:
             model_predict = model_ret
         return model_predict
+
+    @property
+    def data_requirement(self):
+        data_requirement = {
+            "energy": {
+                "ndof": 1,
+                "atomic": False,
+                "must": False,
+                "high_prec": True,
+            },
+            "force": {
+                "ndof": 3,
+                "atomic": True,
+                "must": False,
+                "high_prec": False,
+            },
+            "virial": {
+                "ndof": 9,
+                "atomic": False,
+                "must": False,
+                "high_prec": False,
+            },
+            "atom_ener": {
+                "ndof": 1,
+                "atomic": True,
+                "must": False,
+                "high_prec": False,
+            },
+            "atom_pref": {
+                "ndof": 1,
+                "atomic": True,
+                "must": False,
+                "high_prec": False,
+                "repeat": 3,
+            },
```

Comment:
It seems to me all of these data are only required for energy loss, but not the energy model (if one doesn't train it). It is unclear to me if it is a good way to put these requirements in the energy model.

Reply:
I've made DataRequirementItem on https://github.com/deepmodeling/deepmd-kit/pull/3359/commits/5db7883457df7d7cdc7523a7354d6fb4a5432cfb. And I'm trying to put these requirements into loss definitions.

Reply:
Fix in https://github.com/deepmodeling/deepmd-kit/pull/3359/commits/fdca653a42c9cc0e2e8490d4dfe476416590f80c